MultireferenceReadonly and SingleReferenceReadOnly changes added

4manasa opened the pull request:
Created components to render MultireferenceReadonly and SingleReferenceReadonly

JEAxero commented on src/components/templates/SingleReferenceReadonly/index.ts:
```diff
@@ -0,0 +1,95 @@
+  }
+
+  render(){
+    console.log('pConnn singleref', this.pConn);
```
pConnn has 3 ds in this console.log but I think this should be removed or commented out to remove console clutter.

4manasa replied:
removed the console and updated the PR

JEAxero reviewed (changes requested):
See comment re: moving a console.log. Also, when I try to test this, I'm still seeing some errors like this:

Uncaught (in promise) Error: assignmentID is required parameter
    at Object.openAssignment (constellation-core.5de76a5a.js:2:236472)
    at ListView.clickRowInGrid (index.ts:339:35)
    
So, perhaps we should do a screen share to see if that's a problem beyond the scope of this PR and something that will be fixed later.

4manasa commented:
> See comment re: moving a console.log. Also, when I try to test this, I'm still seeing some errors like this:
> 
> Uncaught (in promise) Error: assignmentID is required parameter at Object.openAssignment (constellation-core.5de76a5a.js:2:236472) at ListView.clickRowInGrid (index.ts:339:35)
> 
> So, perhaps we should do a screen share to see if that's a problem beyond the scope of this PR and something that will be fixed later.

Fixed the above console error and updated the PR

JEAxero reviewed (approved):
Reviewed w/Manasa and looks ok to me.
